src/VkLayer_GF_shader_fuzzer/VkLayer_GF_frame_counter_windows.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I think perhaps this file is wrongly-named (and I guess has the wrong contents) - should it be VkLayer_GF_shader_fuzzer_windows.json?

Reply:
Thanks! The correct file is there. I should have deleted this file!